app/src/main/res/layout/walletcard.xml

```diff
@@ -0,0 +1,386 @@
+<?xml version="1.0" encoding="utf-8"?>
+<androidx.constraintlayout.widget.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"

```

Comment:
Be wary of the many pixel-defined dimensions for UI components. Some of the larger items like the `CardViews` and layouts could have parent- or content-defined heights and widths.

Reply:
Thanks I will take that in note for next time!